### DISCUSSION TITLE
Improve datasets based on subsets of the specimen collection

### DISCUSSION THREAD

**Discussion by jrdh** [NaturalHistoryMuseum/ckanext-nhm#543] [Ideas]
There are a number of datasets on the Data Portal which use URLs of searches across the specimen collection to expose a set of specimens as a group with additional metadata. These are most often used by DCP to expose the specimens digitised in a project together but are also used by others. Here are some examples:

- https://doi.org/10.5519/0014723
- https://doi.org/10.5519/0038559
- https://doi.org/10.5519/83gvsfpc
- https://doi.org/10.5519/0013527

The typical workflow, as far as I understand it, is to search for a project code (or any other search), copy the URL of the search and then create a resource in a dataset with the search URL as the resource URL.

From a brief chat with Laurence the use cases for doing this are:

> - Subcollections – some of our projects are about subcollections that have their own metadata which could go into a collections level description associated with a group of specimens when implemented
> - Documentation – so the specimens can be linked to methodologies in an externally accessible way
> - Awareness, impact and attribution - reporting for impact for funders and collaborators and the wider community who may be interested. Also helps digitisers and the team get recognition for there work as Emu does not record and pass useful attribution data to the Data Portal
> - Findability – sometimes the datasets are supporting a research project or proposal and benefit from additional findability/funding metadata (similar to the subcollections/documentation usecase)

We are keen to make this process easier and more intuitive as the process currently in use is essentially a hack and should be properly supported given it's a legitimate use case. There are also issues around FAIR data with the current process (as flagged in a recent data paper rejection) which given the remit of the Data Portal is to expose FAIR data is something we're very keen to improve.

It would be interesting to hear if there are any more use cases out there or indeed use cases that we may wish to consider. We'll be looking at redesigning/tweaking the user workflow for creating these datasets and looking at how we present them on the Data Portal as well.

Comments welcome!

**Comment by alycejenni** [comment]
Particularly, we're interested in:

1. Why you create these datasets;
2. Why you use these instead of creating a DOI for a search (e.g. https://doi.org/10.5519/qd.crmmojl4);
3. If there are any additional features you'd find useful for these datasets.
